app/analytics/query.ts

```diff
@@ -33,6 +34,7 @@ function accumulateCountsFromRowResult(
         count: number;
         isVisitor: number;
         isVisit: number;
+        bounce: number;
```

Comment:
I think this should be `isBounce`, like the others, for a variety of reasons.

1) It makes it harder to confuse it with `counts.bounces`
2) It matches the naming of the other variables (`isVisitor`, `isVisit`), which is good because they're used in near-identical ways. That association helps communicate intent to readers.
3) It might seem strange that `-1` is a valid value for `isBounce` (normally `is` denotes a boolean), but I think that's fine. If `1` is a bounce, and `0` is not a bounce, then `-1` is a negative bounce.